src/VocabularyManager.UseCases/Services/WordCloner.cs

```diff
@@ -1,6 +1,6 @@
-﻿using DictionaryManager.Shared.Models.DTOs;
+﻿using VocabularyManager.UseCases.DTOs;
 
-namespace DictionaryManagerApp.Services
+namespace VocabularyManager.UseCases.Services
 {
     public class WordCloner
```

Comment:
you can have `Clone` on class object itself. 

btw why do you need clone in first place?

Reply:
I need to copy the object for forms like edit, if the user presses the cancel, they probably don't want to see changes. So, i thought there shouldn't was any references.